.github/workflows/truffleruby-ci.yml

```diff
@@ -0,0 +1,28 @@
+name: RSpec Truffleruby CI
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
You need to add this branch to get it to run here.

Reply:
Actually can we wildcard branches? I'd happily let a `*truffle*` type match remain permanently, then branches named appropriately would run truffle